docs/software-engineering-policies/SecureDevelopment/SecureDevelopmentPolicy.md

```diff
@@ -1,129 +1,140 @@
-* Configuration should also be stored in source control and be subject to the code review process.
+- [ ] All project code intended for delivery should be stored as per the UKHO source control policy.
 
 ### 3rd Party Dependency/Package Management
 
 >_We are responsible for ensuring that 3rd party packages within our solutions are from authorised repositories, have appropriate licensing and are checked for known vulnerabilities._
 
-* Packages must be downloaded from Proget, Nexus, Gerrit (or any ALM authorised tool).
-* The team must use a dependency checking tool such as Dependency Checker (or another ALM authorised tool).
-* The tool must be configured to run as part of either/both a build or release pipeline. Code for production must have passed through this tool before release.
-* The team should decide what the acceptable threshold is to fail a release.
-* The team must provide proof that the release fails when this threshold is breached. Decisions to release even in the presence of a known vulnerability should be recorded and approved by the risk owner.
-
-* The team should agree the schedule for updating the dependency checker.
+- [ ] Packages must be downloaded from a package management tool e.g. Proget or Nexus (or any DDC authorised tool).
+- [ ] The team must use a dependency checking tool such as Dependency Checker (or another DDC authorised tool).
+- [ ] The tool must be configured to run before code is released to production e.g. via a build/release pipeline.
+- [ ] The team should decide what the acceptable threshold is to prevent code from being released to production e.g. via a build/release pipeline.
+- [ ] The team must provide proof that the code is not released to production when this threshold is breached. Decisions to release even in the presence of a known vulnerability should be recorded and approved by the risk owner.
+- [ ] The team should agree on the schedule for updating the dependency checker.
 
 ### SAST Tooling
 
->_Static analysis tooling is required to add an extra layer of verification to developer’s code and to catch vulnerabilities._
+>_Static analysis tooling is required to add an extra layer of verification to the developer’s code around catching vulnerabilities before they are released to production._
 
-* The team must use a static analysis tool which ALM has approved.
-* Each team member must have a basic understanding of interpreting the results.
-* The tool must be configured to run as part of either/both a build or release pipeline. Code for production must have passed through this tool before release.
-* The team should decide what the acceptable threshold is to fail a release.
-* The team must provide proof that the release fails when this threshold is breached.
+- [ ] The team must have a SAST process, either automated or manual via code reviews.
+- [ ] It is strongly recommended that the team use a static analysis tool which DDC has approved.
```

Comment:
Odd to have a recommendation with a tick box.

Reply:
Good point!